cpp/src/io/parquet/reader_impl_chunking.cu

```diff
@@ -253,13 +253,15 @@ struct set_row_index {
   device_span<ColumnChunkDesc const> chunks;
   device_span<PageInfo const> pages;
   device_span<cumulative_page_info> c_info;
+  size_t const max_row;
 
   __device__ void operator()(size_t i)
   {
-    auto const& page            = pages[i];
-    auto const& chunk           = chunks[page.chunk_idx];
-    size_t const page_start_row = chunk.start_row + page.chunk_row + page.num_rows;
-    c_info[i].row_index         = page_start_row;
+    auto const& page          = pages[i];
+    auto const& chunk         = chunks[page.chunk_idx];
+    size_t const page_end_row = chunk.start_row + page.chunk_row + page.num_rows;
+    // if we have been passed a cap, apply it
```

Comment:
```suggestion
    // if we have been past a cap, apply it
```

Reply:
No that really means "passed in".
```suggestion
    // if we have been passed in a cap, apply it
```

Reply:
Oh, I see that now. I thought this was if we are over the cap. Yes, clarification here would be nice.